### DISCUSSION TITLE
How to overwrite clarity breakpoints for a component ?

### DISCUSSION THREAD

**Discussion by gayathrirajendran** [vmware/clarity#6374] [Q&A]
Am trying to use the header or vertical nav's responsive behavior that comes with 768px and below on larger screens too. 
Looking for options to overwrite the breakpoints, can you please let know what the right way to do this is. 
Thanks!

**Comment by nuicore** [comment]
Ever figure out how to do that? I'd like to do the same, I can edit vlr-ui-min.css but adding my own @media in a custom stylesheet doesn't get picked up.
